charts/timeboost/values.yaml

```diff
@@ -0,0 +1,349 @@
+  ## @param auctioneer.podSecurityContext.runAsUser User id for the user
+  ## @param auctioneer.podSecurityContext.fsGroupChangePolicy Policy for the fs group
+  podSecurityContext:
+    fsGroup: 1000
+    runAsGroup: 1000
+    runAsNonRoot: true
+    runAsUser: 1000
+    fsGroupChangePolicy: "OnRootMismatch"
+
+  ## @param auctioneer.securityContext Security context for the container
+  securityContext: {}
+
+  ## @param auctioneer.priorityClassName Priority class name
+  priorityClassName: ""
+
+  ## @param auctioneer.service.type Service type
+  ## @param auctioneer.service.publishNotReadyAddresses Publish not ready addresses
+  service:
+    type: ClusterIP
+    publishNotReadyAddresses: false
+
+  ## @param auctioneer.resources Resources for the container
+  resources: {}
+
+  ## @param auctioneer.nodeSelector Node selector for the pod
+  nodeSelector: {}
+
+  ## @param auctioneer.tolerations Tolerations for the pod
+  tolerations: []
+
+  ## @param auctioneer.affinity Affinity for the pod
+  affinity: {}
+
+  ## @param auctioneer.additionalVolumeClaims Additional volume claims for the pod
+  additionalVolumeClaims: []
+
+  ## @param auctioneer.extraVolumes Additional volumes for the pod
+  extraVolumes: []
+
+  ## @param auctioneer.extraVolumeMounts Additional volume mounts for the pod
+  extraVolumeMounts: []
+
+  ## @param auctioneer.extraPorts Additional ports for the pod
+  extraPorts: []
+
+  ## @param auctioneer.wallet.mountPath Path to mount the wallets
+  ## @param auctioneer.wallet.files Key value pair of wallet name and contents (ethers json format)
+  wallet:
+    mountPath: "/wallet/"
+    files: {} # sequencer-wallet1.json: '{"address": "0x...", "encryptedData": "..." }'
+
+  configmap:
+    ## @param auctioneer.configmap.enabled Enable a configmap for the nitro container
+    enabled: true
+    ## @extra configmap.data See Configuration Options for the full list of options
+    data:
+      ## @param auctioneer.configmap.data.auctioneer-server.auction-contract-address Auction contract address
+      ## @param auctioneer.configmap.data.auctioneer-server.db-directory Database storage directory
+      ## @param auctioneer.configmap.data.auctioneer-server.redis-url Redis URL
+      ## @param auctioneer.configmap.data.auctioneer-server.sequencer-endpoint Sequencer endpoint
+      ## @param auctioneer.configmap.data.auctioneer-server.sequencer-jwt-path Path to seqeuncer JWT
+      ## @param auctioneer.configmap.data.auctioneer-server.wallet.account Wallet account
+      ## @param auctioneer.configmap.data.auctioneer-server.wallet.password Wallet password
+      ## @param auctioneer.configmap.data.auctioneer-server.wallet.pathname Path to wallet
+      auctioneer-server:
+        auction-contract-address: ''
+        db-directory: "/data"
+        redis-url: redis://redis:6379
+        sequencer-endpoint: ws://sequencer:8549
+        sequencer-jwt-path: "/config/jwt.hex"
```

Comment:
Is this the correct location?

Reply:
No, but I can set sane defaults in our actual repo we use for config where they will be correct.